pkg/api/copy_test.go

```diff
@@ -32,24 +32,24 @@ import (
 
 func TestDeepCopyApiObjects(t *testing.T) {
 	for i := 0; i < *fuzzIters; i++ {
-		for _, gv := range []unversioned.GroupVersion{testapi.Default.InternalGroupVersion(), *testapi.Default.GroupVersion()} {
-			f := apitesting.FuzzerFor(t, gv.String(), rand.NewSource(rand.Int63()))
-			for kind := range api.Scheme.KnownTypes(gv) {
-				doDeepCopyTest(t, gv.String(), kind, f)
+		for _, version := range []unversioned.GroupVersion{testapi.Default.InternalGroupVersion(), *testapi.Default.GroupVersion()} {
+			f := apitesting.FuzzerFor(t, version.String(), rand.NewSource(rand.Int63()))
```

Comment:
I know we've gone through this before and you think we don't need to name the variable groupVersion because we've properly typed it now, but I still think naming it groupVersion is better for readability. For example, when reading this line, now I have to find the definition of `version` to know what's contained in it.